golang/pkg/dagent/utils/docker.go

```diff
@@ -785,3 +786,34 @@ func ContainerLog(ctx context.Context, request *agent.ContainerLogRequest) (*grp
 
 	return logContext, nil
 }
+
+func ContainerInspect(ctx context.Context, request *agent.ContainerInspectRequest) (string, error) {
+	cli, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	if err != nil {
+		return "", err
+	}
+
+	prefix := request.Container.Prefix
+	name := request.Container.Name
+
+	cont, err := GetContainerByPrefixAndName(ctx, cli, prefix, name)
+	if cont == nil {
+		return "", &UnknownContainerError{}
```

Comment:
I know it is not your doing, but could you replace the custom struct with a var spawned as
```
var (
ErrUnknownContainer = errors.New("... errtext")
)
```
This is cleaner looking overall, we don't need the custom struct, we don't do anything special.